dcpy/extract/utils.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+import requests
+
+from dcpy.connectors.edm import recipes
+
+
+class Socrata:
```

Comment:
Should this go in the existing Socrata connector?

Reply:
I think so. Was debating waiting for a follow-up but think that makes enough sense to do now